spring/boot2-autoconfigure/src/test/java/com/linecorp/armeria/spring/DeprecatedIpTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.spring;
+
+import java.net.InetAddress;
+import java.util.Collection;
+
+import javax.inject.Inject;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.ServerPort;
+import com.linecorp.armeria.spring.ArmeriaSettings.Port;
+import com.linecorp.armeria.spring.LocalArmeriaPortsTest.TestConfiguration;
+
+/**
+ * Tests for keeping the behavior of {@link Port#getIp()}.
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = TestConfiguration.class)
+@ActiveProfiles({ "local", "deprecatedIpTest" })
+@DirtiesContext
+public class DeprecatedIpTest {
+
+    @SpringBootApplication
+    static class TestConfiguration {}
+
+    @Inject
+    private Server server;
+
+    @Test
+    public void testIpCanBeUsed() {
+        final Collection<ServerPort> serverPorts = server.activePorts().values();
+        for (ServerPort sp : serverPorts) {
+            final InetAddress address = sp.localAddress().getAddress();
+            Assertions.assertThat(address.isAnyLocalAddress() || address.isLoopbackAddress()).isTrue();
```

Comment:
nit: Prefer static import for assertions.
```suggestion
           assertThat(address.isAnyLocalAddress() || address.isLoopbackAddress()).isTrue();
```